testdata/extern_modules/extern-checksum-ebpf.c

```diff
@@ -0,0 +1,37 @@
+#include  "test.h"
+
+/*
+ * This file implements sample C extern function. It contains definition of the following C extern function:
```

Comment:
I don't think you need to justify this as being a "sample" function. I would merge this comment with the next and indicate that this is the C implementation of an extern P4 function.

Reply:
Done.